src/helpers/uploadFiles.ts

```diff
@@ -36,7 +37,7 @@ export async function uploadFiles(
 }
 
 export function getFullURL(baseURL: string, fileName: string) {
-  return `${baseURL}-${fileName.replace(SPACES, "_")}`;
+  return encodeURI(`${baseURL}-${fileName.replace(SPACES, "_")}`);
```

Comment:
As far as I could gather, every `publicUrl` field, which we use to submit files to AWS, gets constructed with this `getFullURL` function, thus the safest way to ensure all URLs are encoded prior to submission is to do it here.

The alternative would be to have a convention where we *must remember* to call `encodeURI` before any submission, which we are sure to forget at some point.

Open to suggestions anyway :+1: